core-sitemaps.php

```diff
@@ -18,3 +18,7 @@
  */
 
 // Your code starts here.
+
+require_once dirname( __FILE__ ) . '/sitemaps-index.php';
```

Comment:
This is the same as the code below but simpler.
```suggestion
require_once __DIR__ . '/sitemaps-index.php';
```